components/fonts/baekmuk/history

```diff
@@ -1 +1 @@
-system/font/baekmuk-ttf@2.2-2018.0.0.0 system/font/truetype/baekmuk
+system/font/baekmuk-ttf@2.2-2018.0.0.0 system/font/baekmuk
```

Comment:
This time bump system/font/baekmuk-ttf@2.2-2018.0.0.0 to system/font/baekmuk-ttf@2.2-2018.0.0.1 and also bump COMPONENT_REVISION.
The same stands for freefont.

Reply:
Ok, I was missing the version increment because it was inconsistent compared to meta-packages/history/history